javatests/google/registry/flows/domain/DomainCreateFlowTest.java

```diff
@@ -437,6 +437,18 @@ public void testSuccess_generalAvailability_ignoresEncodedSignedMark() throws Ex
   }
 
   @Test
+  public void testSuccess_superuser_ignoresEncodedSignedMark() throws Exception {
```

Comment:
Does this test pass in this PR as written?  Looking at the code, I'd expect that this test would only pass if change (1) as proposed in the original email were also made (skipping SMD verification for superuser).  But it looks like the PR presently doesn't change the business logic around that (though see also my comment on that file).

Did we come to a decision about change (1) and whether we're doing it?  Not sure if it was ever established why the superuser can't just not pass the SMD at all in this case.

Also, it might be a little cleaner to have that as a separate PR that changes the business logic and adds this test, now that the rest of the PR to add isSuperuser to the flow custom logic also involves adding FlowMetadata - lots of things going on by now.